config/initializers/secure_headers.rb

```diff
@@ -7,7 +7,7 @@
   :form_action => %w['self'],
   :frame_ancestors => %w['self'],
   :frame_src => %w['self'],
-  :img_src => %w['self' data: www.gravatar.com *.wp.com tile.openstreetmap.org *.tile.openstreetmap.org *.tile.thunderforest.com tileserver.memomaps.de *.openstreetmap.fr],
+  :img_src => %w['self' data: www.gravatar.com *.wp.com tile.openstreetmap.org *.tile.openstreetmap.org *.tile.thunderforest.com tileserver.memomaps.de *.openstreetmap.fr *.global.ssl.fastly.net],
```

Comment:
I'd make this an override in the `embed` method of the export controller rather than enabling it globally.

Reply:
@tomhughes Any examples of this? What is required?

Reply:
Well there is already an override there, it just needs an `img_src` rule adding. You can just add that one source and it will be merged with the global list.

Reply:
I think I properly added the override rule.